### PR TITLE
Ntd1hc/feat/robfw 354/robotlog2db needs append mode

### PR DESCRIPTION
**Pull request by ngoan1608**
Hi Thomas,
Hi Holger,

This PR contains the change for append mode.
Besides, I have also updated README according to Holger's finding in #18 .

Thank you,
Ngoan

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -72,10 +72,11 @@ then you can run it directly as operation system's command.
 Usage
 -----
 
-RobotLog2DB_ tool requires the RobotFramework ``output.xml`` result file(s) and 
+RobotLog2DB_ requires the Robot Framework result file(s) which contains the 
+output result in XML format (default name is ``output.xml``) and 
 TestResultWebApp's database information for importing.
```
Comment (test-fullautomation) [inline README.rst]: Hi Ngoan,
I would expect a link here.
Thank you,
 Thomas

Comment (ngoan1608) [reply README.rst]: Hi Thomas,

I added it in new commit.

Thank you,
Ngoan

**Review by HolQue — COMMENTED** [review]
append mode works; documentation is fine!

**Review by test-fullautomation — APPROVED** [review]
Hi Ngoan,
looks good to me!
Thank you,
 Thomas

**Pull request merged**